### DISCUSSION TITLE
How to create new set of migrations

### DISCUSSION THREAD

**Discussion by FaizulHussain** [skoruba/IdentityServer4.Admin#860] [Q&A] [answered]
I am trying to set a default schema for the tables in the model builder method. However, when I try to add migration I am getting an error. It's a clean project without any other changes. The error message I am getting is the following:

_Your target project 'XXX.IdentityService.Admin' doesn't match your migrations assembly 'XXX.IdentityService.Admin.EntityFramework.SqlServer'. Either change your target project or change your migrations assembly.
Change your migrations assembly by using DbContextOptionsBuilder. E.g. options.UseSqlServer(connection, b => b.MigrationsAssembly("XXX.IdentityService.Admin")). By default, the migrations assembly is the assembly containing the DbContext.
Change your target project to the migrations project by using the Package Manager Console's Default project drop-down list, or by executing "dotnet ef" from the directory containing the migrations project._

Thanks

**Comment by FaizulHussain** [comment, marked as answer]
Got it :)

You need to be in the working directory:
E:\git\XXX.IdentityService\src\XXX.IdentityService.Admin

and then run:
dotnet ef migrations add DefaultSchema --context AdminAuditLogDbContext --project ..\XXX.IdentityService.Admin.EntityFramework.SqlServer\XXX.IdentityService.Admin.EntityFramework.SqlServer.csproj

**Reply by skoruba** [reply]
Hello, sure. Thanks for letting me know that you solved it. 🙂
